simulation/model.py

```diff
@@ -68,6 +68,7 @@ def create_household_agents(
 ) -> Iterator[Household]:
     for household in household_population.itertuples():
```

Comment:
I think this forces `id` to be a column in the dataframe which isnt strictly necessary. I'd probably prefer something like:
```
for household_id, household in household_population.itertuples(index=True)
``` 
or 
```
for household_id, household in enumerate(household_population.itertuples())
```
and remove the `id` column from the dataframe input requirements.

Reply:
I'm confused by your comment. I _want_ `id` to be a column in the dataframe because I use it. It's simple and straightforward.

Reply:
My thinking was that when we extract the rows from the data warehouse there wont be an `id` column there (we will have to use the in-built `pandas` index or something else). 
Perhaps we address this in a future PR when we are explicitly dealing with that integration. 
Happy to close this for now and address later, if you want.

Reply:
We can add an ID column using `farm_fingerprint`, which will create a 64-bit integer fingerprint of the each record. It'll be a handy way to keep track of the households in the input.

Reply:
Closing as we can reproducibly generate an ID in BigQuery using `farm_fingerprint`.